components/automate-cli/cmd/chef-automate/backup.go

```diff
@@ -785,17 +797,43 @@ func runDeleteBackupCmd(cmd *cobra.Command, args []string) error {
 			"Must specify a backup",
 		)
 	}
+	var location string
+	start := 0
+	var newArgs []string
+	var backup []*api.BackupTask
+
+	if strings.Contains(args[0], "/") || strings.Contains(args[0], "\\") {
+		location = args[0]
+		start = 1
+	}
+
+	backup, err := getBackupTask(location)
+	if err != nil {
+		return errors.Errorf("Unable to fetch the backup list with error: %v", err.Error())
+	}
+
+	backupMap := getBackupMapFromBackupList(backup)
 
 	ids, err := idsToBackupTasks(args)
```

Comment:
This should be moved after the id check. The ids should be based on the `newArgs`.

Reply:
we can do that, but then if user sends a address as a first parameter, its getting error out saying we can't parse it.